src/app/account/auth/login/page.tsx

```diff
@@ -0,0 +1,78 @@
+"use client";
```

Comment:
Instead of putting the whole page client side only, how about making the contents here in another component, and marking that has client only?

Reply:
We could call it SignupForm maybe?

Reply:
Maybe only just include the stuff inside AuthCard, so authcard itself gets server rendered while the interactive parts are client only

Reply:
thanks for the review @Aztriltus!
is the reason for not making the whole page csr because csr cascades down the hierachy - meaning if page (top level) is csr, then `<AuthCard/>` will also be forced to do csr even without the `'use client'` directive?

(pinged you on the test account privately)

Reply:
Yep so by making a page client side only, everything inside will also be client side. If we wanna take advantage of server components, we should make sure to minimise the number of client side components where possible!

Reply:
@Aztriltus got it! issue addressed at d2b2d296a274986f8b7c4d5db13f92dd8e97f445